packages/client/src/app/components/modals/bridge/Bridge.tsx

```
@@ -0,0 +1,624 @@
+          signal
+        );
+      } catch (error) {
+        if (signal.aborted) return;
+        if (!isBridgeAbortError(error)) {
+          appendUpdate('error', error instanceof Error ? error.message : 'Bridge failed');
+          if (phaseRef.current !== 'submitted') {
+            setShouldResetOnNextOpen(true);
+          }
+        }
+      } finally {
+        if (signal.aborted) return;
```

Biome (2.4.7)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)